src/native/ffi.ts

```diff
@@ -1,3 +1,5 @@
+import { prepare } from "https://deno.land/x/plug@0.5.2/plug.ts"
```

Comment:
```suggestion
import { dlopen } from "https://deno.land/x/plug@1.0.0-rc.3/mod.ts"
```

[Please use the latest version of plug (1.0.0-rc.3)](https://deno.land/x/plug@1.0.0-rc.3)